app/views/dashboard.erb

```diff
@@ -1,9 +1,66 @@
-<h4 class="text-center">Welcome to Robot World!</h4>
+<h4 id="work" class="text-center">Welcome to Robot World!</h4>
 
-<p><div class="row">
+<p><div id="fuego" class="row">
   <a class="col-md-2 col-md-offset-5 btn btn-success" href="/robots" role="button">Robots</a>
 </div></p>
 
 <p><div class="row">
   <a class="col-md-2 col-md-offset-5 btn btn-default" href="/robots/new" role="button">Create new robot</a>
 </div></p>
+
+<table class="table table-default">
+  <% unless robot_world.robot_ages.values.flatten.empty? %>
+    <tr>
+      <th>Average robot age</th>
+      <th>Population by department</th>
+      <th>Population by city</th>
+      <th>Population by state</th>
+    </tr>
+    <tr>
+      <td><%= robot_world.robot_ages["years"] %> years, <%= robot_world.robot_ages["months"] %> months, and <%= robot_world.robot_ages["days"] %> days old</td>
+      <td><%= robot_world.by_department.first.first %>: <%= robot_world.by_department.shift.last %></td>
+      <td><%= robot_world.by_city.first.first %>: <%= robot_world.by_city.shift.last %></td>
+      <td><%= robot_world.by_state.first.first %>: <%= robot_world.by_state.shift.last %></td>
+    </tr>
+    <% robot_world.robot_ages.each do %>
+      <tr>
+        <td></td>
+        <% unless robot_world.by_department.empty? %>
+          <td><%= robot_world.by_department.first.first %>: <%= robot_world.by_department.shift.last %></td>
```

Comment:
this line looks very familiar to line 21. this is probably a could candidate for a helper method or at least removing this logic from the view. once you start calling more than 2-3 methods on a specific object, you should probably encapsulate that behavior in another method.